drivers/eeprom/Kconfig.eeprom_emu

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2020 Laczen
+# SPDX-License-Identifier: Apache-2.0
+
+config EEPROM_EMULATOR
+	bool "Emulated EEPROM driver"
+	select FLASH
+	help
+	  Enable emulated (on flash) EEPROM support. This mimics an external
+	  EEPROM on a flash partition. The number of writes than can be
+	  performed to the EEPROM is maximized by using a flash area that is
+	  larger the EEPROM area and by storing only changes to the EEPROM data.
```

Comment:
larger than?

Reply:
Yes, and also 'The number of writes tha**t**...'